src/main/steps/tasklistresponse/summary/content.ts

```diff
@@ -723,12 +724,32 @@ const cy: typeof en = (content: CommonContent) => {
 
 export const form: FormContent = {
   fields: {
+    statementOfTruth: {
+      type: 'textAndHtml',
+      textAndHtml: l => `${HTML.H1}${l.statementOfTruth} ${HTML.H1_CLOSE}`,
+    },
+    confirm: {
+      type: 'textAndHtml',
+      textAndHtml: l => `${HTML.STATEMENT_OF_TRUTH_H2}${l.confirm} ${HTML.STATEMENT_OF_TRUTH_H2_CLOSE}`,
+    },
+    warningText: {
+      type: 'warning',
+      label: l => `${l.warningText}`,
+    },
+    submit: {
+      type: 'inset',
+      label: l => `<p class="govuk-body">${l.submit}</p><p class="govuk-body">${l.download}</p>`,
+    },
+    statementOfTruthSubmission: {
+      type: 'textAndHtml',
+      label: l => `${l.statementOfTruthSubmission}`,
+    },
     declarationCheck: {
```

Comment:
Is this working as expected? tested the changes?

Reply:
Yes tested, attached test evidence in the ticket